### ISSUE TITLE
Missing alt text on featured library expert

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#669]
## LIB20230113-09. Missing alt text on the Contact Library Expert photo (Moderate)
**Issue**: The photo of Ayako Yoshimura (Featured Library Expert on the Homepage) includes an empty alt attribute, which indicates the image is decorative. In this case, the image is not purely decorative and should have alt text indicating that it is a photo of Ayako Yoshimura.
[
Full issue description](https://docs.google.com/document/d/15B0iu5qEQOI8BH-P48pgJul1-8A5b9d6Bi6PwoQvlTc/edit?pli=1#heading=h.c4k3ko3uwx8l)
